packages/blade/src/components/BottomNav/_decisions/decisions.md

```diff
@@ -0,0 +1,130 @@
+# BottomNav
+
+Bottom navigation component is a persistent user interface element at the bottom of a mobile app screen, providing quick access to core functionalities through icons and labels.
+
+<img src="bottomnav-title.png" height="300px" />
+
+## Links
+
+- [Figma - Bottom Navigation](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=96521-157514&node-type=instance&m=dev)
+
+## API
+
+<!-- prettier-ignore -->
+```jsx
+import { NavLink } from 'react-router-dom';
+
+<BottomNav>
+  <BottomNavItem 
+    as={NavLink} 
+    isActive={true}
+    title="Payments"
```

Comment:
We can keep this API same as the TabNavItem? 

We use `children` instead of `label`

Reply:
Children we can leave out if we need slot in future no? This one is similar to SideNavLink, ActionListItem, MenuItem, AccordionItem etc.

I think overall what we're following on these item level components is to use `title` as default and use `children` when you want to override it to add slot

Reply:
makes sense, changed topnav to have title.